lib/model/autocomplete.dart

```diff
@@ -363,3 +467,77 @@ class UserMentionAutocompleteResult extends MentionAutocompleteResult {
 // TODO(#233): // class UserGroupMentionAutocompleteResult extends MentionAutocompleteResult {
 
 // TODO(#234): // class WildcardMentionAutocompleteResult extends MentionAutocompleteResult {
+
+class TopicAutocompleteDataProvider extends AutocompleteDataProvider<Topic, TopicAutocompleteQuery, TopicAutocompleteResult> {
+  final PerAccountStore store;
+  final int streamId;
+  Iterable<Topic>? topics;
+  bool isFetching = false;
```

Comment:
```suggestion
  Iterable<Topic>? _topics;
  bool _isFetching = false;
```
I think it is safe to change these to private fields.

Reply:
Yes, thanks!